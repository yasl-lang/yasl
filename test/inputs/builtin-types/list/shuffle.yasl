# Use a seed so that we can reliably test this. Without the seed, this should give a different result each time.
math.seed(100)

const ls = [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13 ]

ls->shuffle()

echo ls

