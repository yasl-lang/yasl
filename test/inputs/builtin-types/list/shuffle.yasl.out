[12, 3, 11, 4, 2, 10, 7, 9, 13, 5, 1, 6, 8]
